.github/workflows/parked.yml

```diff
@@ -0,0 +1,52 @@
+# This CI action is intended for experimental, currently broken, or flaky
```

Comment:
May I ask you to add the name of this CI.

Reply:
After thinking about this more, I think it makes more sense to just remove the `musl` section of the test. This PR can be served as the record in case we want to revert the changes when the test becomes stable again.